packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -222,13 +230,22 @@ const MultiSelect = React.forwardRef(function MultiSelect(
             className={`${prefix}--list-box__invalid-icon ${prefix}--list-box__invalid-icon--warning`}
           />
         )}
+        {selectedItems.length > 0 && (
+          <span
+            className={`${prefix}--visually-hidden`}
+            id={`clear-button-label${id}`}>
+            To clear selection, press Delete
```

Comment:
Should this string be parameterized so it can be translated?

Reply:
ah, that's a good point. Hadn't thought of that!